bin/_mocha

```diff
@@ -18,7 +18,9 @@ const utils = Mocha.utils;
 const interfaceNames = Object.keys(Mocha.interfaces);
 const join = path.join;
 const cwd = process.cwd();
-const getOptions = require('./options');
+const options = require('./options');
```

Comment:
fair warning: changes to this file (and `bin/options.js`) are incompatible with #3556 due to the default test directory defined via other means (there's a `lib/mocharc.json` with Mocha's default settings)